Gemfile.lock

```diff
@@ -62,34 +64,37 @@ GEM
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.4.0)
-    mini_portile2 (2.8.0)
-    nokogiri (1.13.10)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.13.10-x86_64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.13.10-x86_64-linux)
       racc (~> 1.4)
     parallel (1.22.1)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (5.0.1)
-    racc (1.6.1)
+    racc (1.6.2)
     rainbow (3.1.1)
+    rake (13.0.6)
     rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     rexml (3.2.5)
-    rouge (3.30.0)
+    rouge (4.0.1)
     safe_yaml (1.0.5)
-    sassc (2.4.0)
-      ffi (~> 1.9)
+    sass-embedded (1.57.1)
+      google-protobuf (~> 3.21)
+      rake (>= 10.0.0)
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     typhoeus (1.4.0)
       ethon (>= 0.9.0)
-    unicode-display_width (2.3.0)
+    unicode-display_width (2.4.0)
     webrick (1.7.0)
     yell (2.2.2)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-21
```

Comment:
@jbrown-xentity @nickumia-reisys, re: the nokogiri change, I upgraded bundler to 2.x.x and apparently now the platforms are now much more specific.

Reply:
This comes down to building native gems for ARMs and JRuby when available, but I'm not sure of the exact ramifications for our pipeline. Let me do some more digging. Thoughts in the meantime?

Reply:
No thoughts, I understand it, but I guess this is just a bad snyk integration more than it is a problem 😕

Reply:
after reading the release notes, it looks like a snyk problem as they released the same patch to `nokogiri@1.13.10-x86_64-linux`

https://github.com/sparklemotion/nokogiri/releases/tag/v1.13.10

Reply:
this feels resolved